### PR TITLE
add comments and update doc

### PR DESCRIPTION
**Pull request by QuanluZhang**

### CODE REVIEW

File: docs/en_US/Tuner/PPOTuner.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+## PPOTuner
+
+This is a tuner generally for NNI's NAS interface, it uses [ppo algorithm](https://arxiv.org/abs/1707.06347). The implementation inherits the main logic of the implementation [here](https://github.com/openai/baselines/tree/master/baselines/ppo2) (i.e., ppo2 from openai), and is adapted for NAS scenario.
+
+It could successfully tune the [mnist-nas example](https://github.com/microsoft/nni/tree/master/examples/trials/mnist-nas), and has the following result:
+
+![](../../img/ppo_mnist.png)
+
+result of tuning [enas search space](https://github.com/microsoft/nni/tree/master/examples/trials/nas_cifar10) (limited epoch number):
```
Comment (xuehui1991) [inline docs/en_US/Tuner/PPOTuner.md]: Could you point out the search space of this experiment? Could you give more explanation of the performance?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -15,7 +15,7 @@ jobs:
     displayName: 'Install nni toolkit via source code'
   - script: |
       python3 -m pip install flake8 --user
-      IGNORE=./tools/nni_annotation/testcase/*:F821,./examples/trials/mnist-nas/*/mnist*.py:F821
+      IGNORE=./tools/nni_annotation/testcase/*:F821,./examples/trials/mnist-nas/*/mnist*.py:F821,./examples/trials/nas_cifar10/src/cifar10/general_child.py:F821
```
Comment (xuehui1991) [inline azure-pipelines.yml]: why ignore here?

File: docs/en_US/Tuner/PPOTuner.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+PPO Tuner on NNI
+===
+
+## PPOTuner
+
+This is a tuner generally for NNI's NAS interface, it uses [ppo algorithm](https://arxiv.org/abs/1707.06347). The implementation inherits the main logic of the implementation [here](https://github.com/openai/baselines/tree/master/baselines/ppo2) (i.e., ppo2 from openai), and is adapted for NAS scenario.
```
Comment (xuehui1991) [inline docs/en_US/Tuner/PPOTuner.md]: "openai" -> "OpenAI"?

File: docs/en_US/Tuner/BuiltinTuner.md [hunk 4]
```diff
@@ -429,6 +429,13 @@ PPOTuner is a Reinforcement Learning tuner based on PPO algorithm. When you are
 * **trials_per_update** (*int, optional, default = 20*) - The number of trials to be used for one update. This number is recommended to be larger than `trialConcurrency` and `trialConcurrency` be a aliquot devisor of  `trials_per_update`.
 * **epochs_per_update** (*int, optional, default = 4*) - The number of epochs for one update.
 * **minibatch_size** (*int, optional, default = 4*) - Mini-batch size (i.e., number of trials for a mini-batch) for the update. Note that, trials_per_update should be divisible of minibatch_size.
```
Comment (ultmaster) [inline docs/en_US/Tuner/BuiltinTuner.md]: Divisible of? Divisor? Divisible by?

File: docs/en_US/Tuner/PPOTuner.md (hunk 1 above)
Comment (QuanluZhang) [reply docs/en_US/Tuner/PPOTuner.md]: sure

File: azure-pipelines.yml (hunk 2 above)
Comment (QuanluZhang) [reply azure-pipelines.yml]: because it uses nni annotation

File: docs/en_US/Tuner/PPOTuner.md (hunk 3 above)
Comment (QuanluZhang) [reply docs/en_US/Tuner/PPOTuner.md]: fixed, thanks

File: docs/en_US/Tuner/BuiltinTuner.md (hunk 4 above)
Comment (QuanluZhang) [reply docs/en_US/Tuner/BuiltinTuner.md]: fixed, thanks

**Pull request merged**